script/infra-setup

```diff
@@ -21,33 +21,68 @@ echo $CI_PIPELINE_IID
 echo "***************************************"
 ###########
 
-# Deploy OpenEBS
-#mkdir azure
-#cp azoebs_aks_name/config azure/
+#################
+## RUNNER MAIN ##
+#################
 
+echo "Running the litmus test.."
+
+## Create the litmus job for setup OpenEBS
 wget https://raw.githubusercontent.com/atulabhi/litmus/v0.7-RC1/providers/openebs/installers/operator/master/litmusbook/openebs_setup.yaml
 
 kubectl apply -f openebs_setup.yaml
 
 sleep 120
 
-kubectl get po --all-namespaces
+## Obtain the litmus pod name 
+
+litmusPodCmd="kubectl get pod --no-headers -n litmus -o jsonpath='{.items[?(@.metadata.labels.job-name==\"$job_name\")].metadata.name}'"
+litmus_pod=$(eval $litmusPodCmd); retcode=$?
+error_handler $retcode msg="Unable to find litmus test runner pod, exiting" action="exit"
```

Comment:
Include error_handler and task_delimiter function

Reply:
Thanks @gprasath included..